client/my-sites/plans/plan-overview/plan-status/style.scss

```diff
@@ -0,0 +1,57 @@
+.plan-status {
+	margin-bottom: 30px;
+}
+
+.plan-status__info {
+	display: flex;
+	align-items: center;
```

Comment:
We should order these css props alphabetically (We should do this on the whole file)